MoneyManagerModel/InitialData.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using System.Diagnostics;
+
+namespace MoneyManagerModel
+{
+    public static class InitialData
+    {
+        public static List<User> 
+            Users { get; private set; }
+
+        public static List<Asset> 
+            Assets { get; private set; }
+
+        public static List<Category> 
+            Categories { get; private set; }
+
+        public static List<Transaction> 
+            Transactions { get; private set; }
+
+        static InitialData()
+        {
+            using (var sr 
+                = new StreamReader(
+                    $"{AppDomain.CurrentDomain.BaseDirectory}" +
+                    $"/initial_data.json"))
```

Comment:
'$' is unnecessary and path separator sign ('/') is hardcoded, would be better to use platform specific one from Path.DirectorySeparatorChar or use Path.Combine method